style.css

```diff
@@ -688,3 +688,126 @@ nav {
   list-style: none;
   display: flex;
 }
+
+.contact-me {
+  display: flex;
+  flex-direction: column;
+  align-content: center;
+  background-color: #fff;
+}
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  margin-left: 24px;
+  margin-right: 24px;
+}
+
+.contact-me h2 {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  margin-top: 120px;
+  margin-left: 24px;
+  margin-right: 24px;
+  margin-bottom: 58px;
+}
+
+.contact-form input {
+  margin: 24px 0;
+  box-sizing: border-box;
+  padding: 15px 16px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.contact-form textarea {

```

Comment:
- [OPTIONAL] I think it's better to add some `padding` to the textarea to create a space between the text and the border. As not required by the previous reviewer I consider it optional.

<img width="929" alt="Screen Shot 2022-11-08 at 10 41 28 PM" src="https://user-images.githubusercontent.com/47673311/200716545-fc5c4dd1-e63f-4c01-83a9-3fc7855f0a6f.png">